api/account.go

```diff
@@ -0,0 +1,54 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"sort"
+)
+
+// Account represents an account on Section
+type Account struct {
+	ID          int    `json:"id"`
+	Href        string `json:"href"`
+	AccountName string `json:"account_name"`
+	IsAdmin     bool   `json:"is_admin"`
+	BillingUser int    `json:"billing_user"`
+	Requires2FA bool   `json:"requires_2fa"`
```

Comment:
Shouldn't 2FA be used only on the server side?

Reply:
Yep, but this is just repeating attributes that the API exposes.

Reply:
The GraphQL schema should not expose fields that we do not need.

Reply:
Also, if the input JSON has more fields than the struct, that is fine, they will be ignored.